docs/src/app/(docs)/getting-started/tanstack-start/page.mdx

```diff
@@ -130,25 +130,31 @@ export const UploadDropzone = generateUploadDropzone<OurFileRouter>();
 
 <Tabs tabs={["Tailwind", "Not Tailwind"]}>
   <Tab>
-      Wrap your Tailwind config with the `withUt` helper. You can learn more about our
-      Tailwind helper in the ["Theming" page](/concepts/theming#theming-with-tailwind-css)
+    Wrap your Tailwind config with the `withUt` helper. You can learn more about our
+    Tailwind helper in the ["Theming" page](/concepts/theming#theming-with-tailwind-css)
 
-      ```tsx
-      import { withUt } from "uploadthing/tw";
+    ```tsx
+    import { withUt } from "uploadthing/tw";
 
-      export default withUt({
-        // Your existing Tailwind config
-        content: ["./src/**/*.{ts,tsx,mdx}"],
-        ...
-      });
-      ```
+    export default withUt({
+      // Your existing Tailwind config
+      content: ["./src/**/*.{ts,tsx,mdx}"],
+      ...
+    });
+    ```
 
   </Tab>
   <Tab>
-    Include our CSS file in the root layout to make sure the components look right!
+    Include our CSS in your root route to make sure the components look right!
 
-    ```tsx
-    import "@uploadthing/react/styles.css";
+    ```tsx {{ title: "app/routes/__root.tsx" }}
+    // @ts-expect-error
```

Comment:
?url imports should be fine if you have vite/client types in your project

Reply:
Vite is not explicitly installed in `tanstack/start` projects and adding `/// <reference types="vite/client" />` results in `Cannot find type definition file for 'vite/client'.`. Might be doing something wrong.

But following [Tanstack's getting started](https://tanstack.com/router/latest/docs/framework/react/start/getting-started) results in setup where `?url` imports result to error. They also do `// @ts-expect-error` in their [examples](https://github.com/TanStack/router/blob/main/examples/react/start-basic/app/routes/__root.tsx#L12). (Some examples don't have `// @ts-expect-error`, but I cloned the repo and typescript does complain.)

Might be "alpha release" thing?

Reply:
But I guess it's mostly for editors. Because even without `// @ts-expect-error`, the build passes. Just VS code gives red squiggly lines.